src/openMVG/cameras/Camera_IO.hpp

```diff
@@ -24,15 +24,22 @@ static bool save(
   const PinholeCamera & cam)
 {
   if (stlplus::extension_part(scameraFile) != "bin")
-    return false;
-
-  const Mat34 & PMat = cam._P;
-  std::ofstream file( scameraFile.c_str(), std::ios::out|std::ios::binary);
-  file.write((const char*)PMat.data(),(std::streamsize)(3*4)*sizeof(double));
-
-  bool bOk = (!file.fail());
-  file.close();
-  return bOk;
+  {
+    return false;    
+  }
+  else 
```

Comment:
I agree, I mainly use clang-tidy tool and it produces a warning here if no else branch is present. I think it's a false positive and I did not make any modification like that elsewhere with the same warning.